XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1914,6 +1915,14 @@ private static string UpdatesFriendlyName(string propertyName)
             return Core.PropertyManager.FriendlyNames.GetString(string.Format("Label-{0}", propertyName)) ?? propertyName;
         }
 
+        private static string UpdatesFriendlyNameAndVersion(Pool_update update)
+        {
+            var friendlyName = UpdatesFriendlyName(update.Name);
+            if (string.IsNullOrEmpty(update.version))
+                return friendlyName;
+            return string.Format(Messages.SUPP_PACK_DESCTIPTION, friendlyName, update.version);

```

Comment:
These changes look ok to me, but since you're here could you please submit one more commit to correct "DESCTIPTION".